src/docs/sdk/sessions.mdx

```diff
@@ -145,6 +145,77 @@ The following fields exist:
 >   agent of the user that caused the session. This data is not persisted but
 >   used for filtering.
 
+## Session Aggregates Payload
+
+<Alert title="Experimental Status" level="info">
+
+The `"sessions"` envelope item is still unfinished and considered experimental.
+It is currently only supported on the `sentry.io` installation and is subject
+to heavy usage-limits.
+
+Currently, the sum of all session counts must not exceed _100_. This limitation
+will be lifted in the future when this feature will become stable.
+
+</Alert>
+
+Especially for _request-mode_ sessions (see below), it is common to have
+thousands of requests, and thus sessions, per second.
+
+Under the assumption that these sessions will be short-lived, and tracking their
+duration is not desired, these sessions can be aggregated together on the
+SDK side, before they are sent to Sentry.
+
+An SDK should aggregate closed sessions and group them by their `started` time
+and `distinct_id`. These groups will be sent as `sessions` envelope item.
+It consists a JSON payload that looks roughly like this:
+
+```json
+{
+  "aggregates": [
+    {
+      "started": "2020-02-07T14:16:00Z",
+      "exited": 123
+    },
+    {
+      "started": "2020-02-07T14:16:00Z",
+      "did": "optional distinct user id",
+      "exited": 12,
+      "errored": 3
+    }
+  ],
+  "attrs": {
+    "release": "release name",
+    "environment": "environment name",
+    "ip_address": "optional user ip address for filtering",
+    "user_agent": "optional user agent for filtering"
+  }
+}
+```
+
+Note that this must be enclosed in an envelope. So the full envelope looks
+something like this:
+
+```json
+{}
+{"type":"sessions"}
+{"aggregates":[...]}
+```
+
+`aggregates`
```

Comment:
We usually use definition lists for formatting these. There is a markdown syntax that is unfortunately broken in Gatsby, so you will have to resort to HTML at the moment:

```
<dl>
<dt><code>aggregates</code></dt>
<dd>

Markdown here.

</dd>
<dl />
<dd />
...
</dl>
```

Note the empty lines around the markdown content.

Reply:
Interesting. I looked at the other docs and they seem to use the definition-list syntax, so I adapted the session docs as well.